### PR TITLE
Update dependencies to revision 78dfa07

### PR DESCRIPTION
**Pull request by azoyan**

### CODE REVIEW

**Review by bezrazli4n0 — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -1,32 +1,58 @@
...
+nft-io = { path = "io" }
+nft-state = { path = "state" }
+primitive-types = { version = "0.12.1", default-features = false }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.7" }
+gear-lib-derive = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.7" }
+sp-core-hashing = { version = "8.0.0", default-features = false }
+hashbrown = "0.13"
+hex-literal = "0.4"
+sp-core = { git = "https://github.com/gear-tech/substrate.git", rev = "36699c4478ce8ae05517bb42a1b115403313e083" }
+tokio = { version = "1", features = ["full"] }
```
Comment (fluiderson) [inline Cargo.toml:58]:
```suggestion
tokio = "1"
```

File: Cargo.toml [hunk 2]
```diff
@@ -1,32 +1,58 @@
...
+authors = ["Gear Technologies"]
 
 [workspace]
 members = ["state"]
 
-[features]
-binary-vendor = []
+[workspace.dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], rev = "78dfa07" }
+gmeta = { git = "https://github.com/gear-tech/gear", rev = "78dfa07", features = ["codegen"] }
```
Comment (fluiderson) [inline Cargo.toml:45]:
```suggestion
gmeta = { git = "https://github.com/gear-tech/gear", rev = "78dfa07" }
```
Please enable that only in the `state` crate's `Cargo.toml`.

File: Cargo.toml [hunk 3]
```diff
@@ -1,32 +1,58 @@
...
+license = "MIT"
+authors = ["Gear Technologies"]
 
 [workspace]
 members = ["state"]
 
-[features]
-binary-vendor = []
+[workspace.dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], rev = "78dfa07" }
```
Comment (fluiderson) [inline Cargo.toml:44]:
```suggestion
gstd = { git = "https://github.com/gear-tech/gear.git", rev = "78dfa07" }
```
Same here. Enable where it's necessary.

**Review by fluiderson — APPROVED** [review]
Please add `rust-toolchain.toml`.

**Pull request merged**